index.js

```diff
@@ -622,14 +622,14 @@ function base64Slice (buf, start, end) {
   }
 }
 
-function decodeCodePointsArray (array) {
-  // Based on http://stackoverflow.com/a/22747272/680742, the browser with
-  // the lowest argument limit is Chrome, with 0x10000 args.
-  if (array.length < 0x10000) return String.fromCharCode.apply(String, array)
+function decodeCodePointsArray (buf) {
+  var len = buf.length
+
+  if (len <= 0x10000) {
```

Comment:
Should just merge all this code into `binarySlice`?
I wonder if the manual string concatenation is slower than `String.fromCharCode.apply(String, buf.slice(start, end))`.
@nolanlawson reckon you could find out?